src/types/onyx/CancellationDetails.ts

```diff
@@ -0,0 +1,29 @@
+import type {CancellationType, FeedbackSurveyOptionID} from '@src/CONST';
+import type * as OnyxCommon from './OnyxCommon';
+import type PrivateSubscription from './PrivateSubscription';
+
+/** Cancellation details model */
+type CancellationDetails = {
+    /** Cancellation date */
+    cancellationDate?: string;
+
+    /** Cancellation reason */
+    cancellationReason?: FeedbackSurveyOptionID;
+
+    /** Cancellation type (manual/automatic/none) */
+    cancellationType?: CancellationType;
```

Comment:
this as well?

Reply:
Same here

Reply:
I don't think this should be optional. Lets see if we can refactor 👍.